ios/Exponent/Versioned/Core/UniversalModules/EXNotifications/EXScopedNotificationBuilder.m

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import "EXScopedNotificationBuilder.h"
+
+@interface EXScopedNotificationBuilder ()
+
+@property (nonatomic, strong) NSString *experienceId;
+
+@end
+
+
+@implementation EXScopedNotificationBuilder
+
+- (instancetype)initWithExperienceId:(NSString *)experienceId
+{
+  if (self = [super init]) {
+    _experienceId = experienceId;
+  }
+  
+  return self;
+}
+
+- (UNNotificationContent *)notificationContentFromRequest:(NSDictionary *)request
+{
+  UNMutableNotificationContent *content = [[super notificationContentFromRequest:request] mutableCopy];
```

Comment:
Huh… I would be afraid to do `UNNotificationContent mutableCopy` (I know we know that `notiicationContentFromRequest` returns a mutable content but…

➡️ Maybe it should return an `UNMutableNotificationContent` everywhere?